frontend/src/common/Title.tsx

```diff
@@ -0,0 +1,13 @@
+import Head from 'next/head'
+
+export interface TitleProps {
+  title: string
```

Comment:
Consider renaming to `text` so we don't have a prop with the same name as the component